css/queries.css

```diff
@@ -0,0 +1,154 @@
+/* Media queries */
```

Comment:
- [ ] Kindly make sure the mobile version is not overlapping. the content of it is supposed to be 100% the max width of the screen size. We do not need to scroll to the right to see the all content.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200887780-80891d72-d3ac-456b-a43d-b64bdee834a9.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200887971-3df91451-3205-44d9-8d09-2c0ee868d844.png)